document/CoreFormat.mdk

```diff
@@ -1649,7 +1689,6 @@ but `name` is not equal to the property name, so it cannot be omitted.
 A `logicalLocationComponent` object should contain a property named `kind`, whose value shall be one of the following strings,
 if any of those strings accurately describes the construct identified by this object:
 
-*   `declaration`
```

Comment:
Removed this per our discussion. It may come back as a property on a "logicalLocation" object, or it may show up in some other way, but it's not going to the a valid value for `logicalLocationComponent.kind`.